tagging_pipeline/load.py

```diff
@@ -0,0 +1,135 @@
+from os import environ
+from datetime import datetime
+
+import pandas as pd
+import psycopg2
+from psycopg2 import extras
+import spacy
+
+from dotenv import load_dotenv
+
+CURRENT_TIMESTAMP = datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+CSV_FILE = f'{CURRENT_TIMESTAMP}.csv'
+
+
+def db_connection() -> psycopg2.extensions.connection | None:
```

Comment:
can simplify this type hint by doing

`from psycopg2.extensions import connection`

and then just `connection` can be used in the type hint